css/main.css

```diff
@@ -4941,3 +4940,6 @@ body.tg-comingsoonpage .tg-foorterbar .tg-socialicons{padding:0;}
   color: white;
 }
 
+.height-100vh {
```

Comment:
What is the use of this class? Can you please explain? I don't see it being used anywhere. If not needed please remove.